### PR TITLE
media-gfx/gimp: 2.10.20 version bump

### PR DESCRIPTION
**Pull request by band-a-prend**
Update dependencies: media-libs/babl-0.1.78, media-libs/gegl-0.4.24

Closes: https://bugs.gentoo.org/726176

### CODE REVIEW

**Comment by DarthGandalf** [comment]
Shouldn't babl and gegl be in separate commits?

File: media-gfx/gimp/gimp-2.10.20.ebuild [hunk 1]
```diff
@@ -0,0 +1,224 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
```
[the remark this answers is not in the document]
Comment (band-a-prend) [reply media-gfx/gimp/gimp-2.10.20.ebuild:5]: No. Gimp-2.10.x is only python-2.7 currently.
The gimp-9999 and future gimp-2.99.2 (and then gimp-3) support python-3.

File: media-gfx/gimp/gimp-2.10.20.ebuild [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+WANT_AUTOMAKE=
+
+inherit autotools gnome2 python-single-r1 toolchain-funcs virtualx
+
+DESCRIPTION="GNU Image Manipulation Program"
+HOMEPAGE="https://www.gimp.org/"
+SRC_URI="mirror://gimp/v2.10/${P}.tar.bz2"
+LICENSE="GPL-3 LGPL-3"
+SLOT="2"
+KEYWORDS="~alpha amd64 arm64 ~hppa ~ia64 ppc64 ~x86"
```
[the remark this answers is not in the document]
Comment (band-a-prend) [reply media-gfx/gimp/gimp-2.10.20.ebuild:16]: Sorry, forgot that it's was stabilized

**Comment by band-a-prend** [comment]
Split first commit into atomic commits for version bump of dependencies.

File: media-gfx/gimp/gimp-2.10.20.ebuild (hunk 1 above)
[the remark this answers is not in the document]
Comment (band-a-prend) [reply media-gfx/gimp/gimp-2.10.20.ebuild:5]: The media-plugins/gimp-resynthesizer depends on media-gfx/gimp[python] while last depends on dev-python/pygtk especially gimp-python-console depends on pygtk while  gimp-resynthesizer  just has some python scripts to run.

[the remark this answers is not in the document]
Comment (band-a-prend) [reply media-gfx/gimp/gimp-2.10.20.ebuild:5]: Should I finaly drop `dev-python/pygtk` dependency this time? https://qa-reports.gentoo.org/output/gentoo-ci/bc2d129407/output.html#media-gfx/gimp

2.10.20 | DeprecatedPkg | deprecated DEPEND: >=dev-python/pygtk-2.10.4:2[python_targets_python2_7(-)]
-- | -- | --
  | DeprecatedPkg | deprecated RDEPEND: >=dev-python/pygtk-2.10.4:2[python_targets_python2_7(-)]

`media-plugins/gimp-resynthesizer` depends on gimp[python] and I could try to modify it's ebuild to conditionally `python? ( media-gfx/gimp[python(-)?] )` as `media-plugins/gimp-resynthesizer` plugin python scripts are part of it and provide some tools without python support.

I then shall close re-opening issue about this as `can't fix` only providing gimp-2.10.20[python] in bugzilla attachment for local overlay.

**Comment by band-a-prend** [comment]
Should I ignore this CI breakage or it expects I need some ebuild changes (use flag mask etc.?)

**Comment by a17r** [comment]
You can only ignore it for now.

**Pull request closed without merging**